EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+
+At this rate, the ETH inflation rate will increase to  5,554,683 ETH/yr. (4,982,250+572,433 =5,554,683). This yields an annual inflation rate of  ~ 5.00% (5,554,683/110,875,500 * 100). At this rate, Ethereum’s inflation will be 2.81x (5.00/1.78) higher than bitcoin.
+
+In order to calculate the block subsidy required to achieve inflation parity with Bitcoin, we must first back out our estimations for ETH2.0 issuance in order to determine the maximum annual POW reward.  
+
+1,973,583.9 - 572,433 = 1,401,150.9 Max Annual PoW Rewards.
+
+Now, we calculate the max daily reward.
+
+1,401,150.9 / 365 = 3839 Max Daily PoW Rewards.
+
+With a targeted block time of 13s, there are approximately 60*60*24 / 13.1 = 6,646 blocks per day.
+
+3839/6646 = 0.5776 ETH per block.
+
+According to Etherscan, uncle rewards are responsible for approximately 5% of the total daily reward emission. Therefore, the base block subsidy should be 0.5776 * 0.95 = 0.549.
+
+Thus, we arrive at a rate of 0.55 ETH base block reward to match Bitcoin’s inflation rate. 
+
+We further note however, that the transaction fee market for Ethereum has risen sharply this year. As of August 11 2020, at the time of publishing this EIP, the fees from transactions make up almost 80% the current block rewards (2ETH). Thus, even if the block reward was set to zero, miners would still earn 1.8ETH from transaction fees per block.  Due to a robust and thriving fee market, we prepose to round to round down our block reward calculation from 0.55 to 0.5, which means (at the time of writing) miners would earn 1.8ETH in fees + 0.5ETH block reward, for a total of 2.3ETH per block. 
+
+
+## Backwards Compatibility
+All nodes must be upgraded to reflect the change in the block reward.
+
+
+
+## Security Considerations
+Changing the block reward is purely an economic action, requiring only a change to the block reward parameter.
+
+Economically speaking, a block reward that is too low may result in low miner participation due to insufficient financial incentives.  This may result in a decrease of network security that may put PoW blockchains at risk of 51% re-org attacks. 
+
+In this EIP we have laid out our economic evidence that reducing the block reward to 0.5 ETH would not negatively affect security by comparing it to other blockchains with larger market capitalizations.  Furthurmore, we note that Miners' expenses for housing and electricity are priced in FIAT terms (no electrical utilities in the in the world currently charge for Kilowatt-Hours in ETH). Therefore, miners make their mining decisions based on FIAT revenues they expect to earn. In $USD terms, miners would still be paid more than they were 3 months ago if this EIP were accepted (and still more than the 12 month average).  Thus, with this EIP, mining security economics are still better than they were 3 months ago when the price of ETH was half of what it is now, and the transaction fees were a fraction of their current value. 
```

Comment:
> (no electrical utilities in the in the world currently charge for Kilowatt-Hours in ETH)

Proof?

Reply:
> A block reward that is too high is inefficient and negatively impacts the ecosystem economically.

Proof ?

Reply:
Please direct your comments to the discussion forum: https://ethereum-magicians.org/t/eip-2878-block-reward-reduction-to-0-5-eth/

Reply:
Ridiculous proposal at the end of PoW. 
Dishonest to say mining will not be affected or security either.
Why should we compare inflation rates to BTC?
What is the problems for miners to profit is the whole chain integrity rellies on them?
We won't be worrying about inflation if DeFi craze combined with lower processing power in the chain start making bottlenecks.
It's time to have the best version of ETH 1.0, and not jeopardize it just before the giant leap ETH 2.0 will mean.
This EIP is at least irresponsible.

Reply:
Since this is a highly contentious proposal, I'm going to be deleting comments that should go in the discussion-to thread.  Please keep discussion in the appropriate location, PR comments are for specific changes/feedback to wording, layout, specification details, etc. not discussion about the merit of the EIP.

Reply:
> > (no electrical utilities in the in the world currently charge for Kilowatt-Hours in ETH)
> 
> Proof?

It is nearly impossible to prove a negative.  After an exhaustive review, we are unaware of any electrical utilities globally that take payment in *fixed* prices per kWh in ETH terms.  This would be insane for utilities to do, as they would take a lot of risk in the price volatility of ETH.

We will not retract this claim unless you can prove otherwise.  Thank you.

Reply:
> > A block reward that is too high is inefficient and negatively impacts the ecosystem economically.
> 
> Proof ?

This is a claim we stand by.  Over-paying for security is inefficient.  Inefficiencies are not good for the ecosystem (or almost anything in general).  Paying too much for something is an economic inefficiency, and impacts the paying negatively. 

If you can prove that over-paying for goods or services is a positive thing, we will consider revising this claim.

Reply:
> > (no electrical utilities in the in the world currently charge for Kilowatt-Hours in ETH)
> 
> Proof?

Proving a negative is nearly impossible.  This is a claim we stand by.   After doing an extensive search, we did not find any electrical utilities in the world that charge for kWh in fixed prices per ETH.

We will not retract this claim from the PR unless proven otherwise. Thank you.

Reply:
My personal opinion is that an EIP that significantly changes the monetary policy should undergo formal analysis (e.g. see EIP-1559) to understand the longer term consequences.